test/avt/detect-ws.xspec

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<x:description
+	xmlns:mirror="x-urn:test:mirror"
+	xmlns:myv="http://example.org/ns/my/variable" 
+	xmlns:x="http://www.jenitennison.com/xslt/xspec">
+
+	<x:scenario label="Detect if reformatting interfered with whitespace characters that are important">
+		<!-- Use unparsed-text instead of doc() to avoid having
+			attribute value normalization convert the tabs and
+			line feeds to spaces. -->
+		<x:call function="mirror:param-mirror">
+			<x:param select="$x:xspec-uri => unparsed-text()" />
+		</x:call>
+		<!-- Use || to avoid tokenizing on the tokenize() function call itself. -->
+		<x:variable name="myv:function-calls"
```

Comment:
Is it still necessary to use `||`? I ran the tests without it, and it appeared to work.

Reply:
You are right. I think I initially ran this test directly instead of via import from a parent XSpec file. Now that this file is exposed only via `x:import`, the tokenization function call does not need to be shielded from itself.

Addressed via 267021f17a29738316c85c20cf2dcfe5d7392739.